test/spec/modules/teadsBidAdapter_spec.js

```diff
@@ -211,9 +211,15 @@ describe('teadsBidAdapter', () => {
       if (ttfbExpectedV2) {
         expect(payload.timeToFirstByte).to.deep.equal(ttfbExpectedV2.toString());
       } else {
-        const ttfbExpectedV1 = performance.timing.responseStart - performance.timing.requestStart;
-
-        expect(payload.timeToFirstByte).to.deep.equal(ttfbExpectedV1.toString());
+        const ttfbWithTimingV1 = performance &&
+            performance.timing.responseStart &&
+            performance.timing.requestStart &&
+            performance.timing.responseStart >= 0 &&
+            performance.timing.requestStart >= 0 &&
```

Comment:
these can't be zero because of the lines above.  Not a big deal but could be `>` rather than `>=`.

Reply:
You are right, updated it test side and adapter side. Thank you for the review